### ISSUE TITLE
Malformed file warning if Rmd contains raw attribute chunk

### ISSUE THREAD

**Issue by stefanocoretta** [r-lib/lintr#903]
Hello!

It seems as if lintr returns the following warning if the Rmd file contains a code chunk with a Pandoc [raw attribute](https://pandoc.org/MANUAL.html#extension-raw_attribute).

```r
1:1   error   Failed to run diagnostics: callr subprocess failed: Malformed file! ​lintr
```

The following MWE raises the warning.

    ---
    title: Test
    ---
    
    Example Rmd with raw attribute.
    
    ```{=latex}
    \textit{test}
    ```

**Comment by AshesITR** [comment]
Thanks for reporting this, I think it might have been resolved by #575 in the development version of lintr.
I can't reproduce on the current master branch. Can you try it out? Thanks!

**Comment by MichaelChirico** [comment]
Seems working on current devel; closing

```
s = '---
title: Test
---

Example Rmd with raw attribute.

```{=latex}
\textit{test}
```'
writeLines(s, tmp <- tempfile(fileext='.Rmd'))
lintr::lint(tmp)
```
